packages/kotti-ui/source/kotti-field-password/types.ts

```diff
@@ -12,7 +12,9 @@ export module KottiFieldPassword {
 		NEW = 'new-password',
 	}
 	export const propsSchema = KottiField.propsSchema
-		.merge(KottiField.potentiallySupportedPropsSchema)
+		.merge(
+			KottiField.potentiallySupportedPropsSchema.omit({ autoComplete: true }),
```

Comment:
Autocomplete should be supported on passwords, as `new-password`, `current-password`, and `one-time-code` are potentially helpful for password managers to understand password fields

Reply:
`KtFieldPassword` already supports `new-password` and `current-password`. Added `one-time-code`.